### ISSUE TITLE
More named colour fixes

### ISSUE THREAD

**Issue by niuserre** [mouse0270/lib-themer#37]
Loving the mod, I know you said it's for other modules to use but all I really want to do was improve my colours and it's wonderful for that. Not totally sure how this works so no worries if this is just me trying to do awkward combinations.

Nothing seems to change this scrollbar colour (with or without Colorise ticked):
![image](https://user-images.githubusercontent.com/1446741/211219674-034b4f80-6263-4b5c-839e-a16d5dc26572.png)
CSS via dev tools says `Pseudo ::scrollbar-thumb element` and points to style.css:474 which is:
```
::-webkit-scrollbar-thumb {
  outline: none;
  border-radius: 3px;
  background: #782e22;
  border: 1px solid var(--color-border-highlight);
}
:root {
  scrollbar-width: thin;
  scrollbar-color: #782e22 #111;
}
```

And this one only happens using the Colorise options specifically, the search in the sidebar gets dark on dark:
![image](https://user-images.githubusercontent.com/1446741/211220021-bdb175f5-0a58-438a-a727-a5810b776992.png)

Which is related to `foundry-vtt-colors.css:196` and `foundry-vtt-colors.css:21`:
```
.directory .directory-header .header-search input {
    background: rgba(0, 0, 0, 0.05);

input[type="text"], input[type="number"], input[type="password"], input[type="date"], input[type="time"], input[type="search"], input[type="color"], input[type="color"][data-edit], select {
    ~~background: rgba(0, 0, 0, 0.05);~~
    border-color: var(--app-dialog-background-shaded);
    color: var(--app-dialog-background-contrast);
}
```

I have used the custom CSS module to do `.header-search input {color:#ffffff;}` which is good enough for me.

**Comment by KelSolaar** [comment]
This worked for me:

![Uploading image.png…]()

**Comment by mouse0270** [comment]
Yeah, the colorized options was just kinda me quickly showing how to use the themer. I am happy to patch stuff like this though, because this should be updated on libThemer

**Comment by niuserre** [comment]
> Yeah, the colorized options was just kinda me quickly showing how to use the themer. I am happy to patch stuff like this though, because this should be updated on libThemer

The colorising is awesome and I don't expect perfection. I'm happy to mention things I notice so you can choose what to do with the feedback 💜 

> ![image](https://user-images.githubusercontent.com/99779/211446434-459e3533-d4b2-4cff-93f9-66d575545e91.png)

I believe that's just changing the outline. In my screenshot you can see the border is turquoise but the middle is brown. Bit hard to see but I think this screenshot still has the middle as brown, just with a red outline. 🙂
